modules/rosa-upgrading-4_7_to_4_8_prerequisites.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * rosa_upgrading/rosa-upgrading-sts.adoc
+
+[id="rosa-upgrading-4-7-to-4-8-prerequisites_{context}"]
+= Prerequisites for upgrading from 4.7 to 4.8
+
+This section details the steps that are required before upgrading a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS) from version 4.7 to 4.8. Before upgrading, you must meet the following prerequisites:
```

Comment:
"This section" is unnecessarily self-referential. From the ISG, "Global audiences" > "Style":
> Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes".

Obligitory sugg: "You must perform the following steps before upgrading..."  Up to you.

I suppose this is a nit, but "...you must meet the following prerequisites:" seems to be layering prereqs--aren't the bullets the actions that are covered in the procedure?  There's a block title for `.Prerequisites`, but it follows the two bullets that are identified as prereqs.  Just pointing it out as a second set of eyes.  Perhaps "Before upgrading, you must perform the following procedure to meet the following upgrade requirements:"

Reply:
Thanks Mike for highlighting this. I have reduced the paragraph down to one sentence that resolves all of this:

"You must meet the following requirements before upgrading a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS) from version 4.7 to 4.8:"